build/reference/9082ConflictingMethodModifers.txt

```diff
@@ -0,0 +1,16 @@
+W082 Conflicting Method Modifiers
+Errors and Warnings 51-99
+noreferences
+
+@@description
+
+<h2>Umple sematic warning when do activities, exit actions, outgoing transitions, and/or nested state machines are removed by the compiler from final states.</h2>

```

Comment:
This block is leftover from error 72 and is not correct in this file. Please change so it is correct.